tools/cldr-apps/js/src/esm/cldrTable.mjs

```diff
@@ -921,7 +921,7 @@ function addVitem(td, tr, theRow, item, newButton) {
       cldrText.get("voteInfo_baseline_desc")
     );
   }
-  checkLRmarker(choiceField, item.value);
+  checkLRmarker(choiceField, displayValue);
```

Comment:
this is an old bug, `displayValue` is updated above to be the inherited value instead of cldrSurvey.INHERITANCE_MARKER